pom.xml

```diff
@@ -467,6 +567,16 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
```

Comment:
we can add goal `analyze-only` to standard build

Reply:
In that case, I will also add `failOnWarning` to ensure that findings are apparent.